libs/newsletters-data-client/src/lib/newsletter-data-type.ts

```diff
@@ -32,10 +33,19 @@ export const singleThrasherLocation = z
 	.optional();
 export type SingleThrasherLocation = z.infer<typeof singleThrasherLocation>;
 
+export const readMoreSectionSchema = z
+	.object({
+		subheading: nonEmptyString().describe('read more subheading'),
+		wording: nonEmptyString().describe('read more wording'),
+		url: z.string().url().describe('read more url'),
```

Comment:
did we want this to be .optional() as well?

Reply:
yes - that should be optional!